fitlins/interfaces/bids.py

```diff
@@ -113,9 +113,11 @@ def _run_interface(self, runtime):
 
 
 class LoadLevel1BIDSModelInputSpec(BaseInterfaceInputSpec):
-    bids_dirs = InputMultiPath(Directory(exists=True),
-                               mandatory=True,
-                               desc='BIDS dataset root directories')
+    bids_dir = Directory(exists=True,
+                         mandatory=True,
+                         desc='BIDS dataset root directory')
+    preproc_dir = Directory(exists=True,
+                            desc='Optional preprocessed files directory')
```

Comment:
Maybe:

```Python
confounds_dir = InputMultiPath(Directory(exists=True),
                               desc='Derivatives directories containing confounds files')
```

Reply:
Unless you've already worked on this, let's leave it for another PR. This gets where we need to be in separating the directories.